ivy/functional/frontends/torch/linalg.py

```diff
@@ -27,3 +28,9 @@ def slogdet(input, *, out=None):
 @to_ivy_arrays_and_back
 def matrix_power(input, n, *, out=None):
     return ivy.matrix_power(input, n, out=out)
+
+
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes({"1.11.0 and below": ("float16",)}, "torch")
+def cross(input, other, *, dim=-1, out=None):
+    return torch_frontend.cross(input, other, dim, out=out)
```

Comment:
You would need to use Ivy functions here, rather than torch functions.

Reply:
Hi,

Thanks for the review. I have implemented this function according to the instructions provided in the ToDo list [here](https://github.com/unifyai/ivy/issues/7479#issue-1469563462):

> Note: If the Linear Algebra function to be implemented has identical behavior to another PyTorch function, you should simply keep an alias in the linalg.py file rather than creating a duplicate implementation.

Here, it is explicitly mentioned **not** to provide a duplicate implementation of an already implemented PyTorch function that has an identical behavior, in my case that is `torch.cross`, but just use it to define an alias in `../torch/linalg.py` as I've done so. The example provided in that ToDo list highlights exactly what I've implemented. Can you kindly cross verify with the guidelines we were provided and what I've implemented? 🙂 Thanks

Reply:
I have checked and this is the correct protocol :)